tools/go_generics/globals/globals_visitor.go

```diff
@@ -47,6 +47,9 @@ type globalsVisitor struct {
 
 	// scope is the current scope as nodes are visited.
 	scope *scope
+
+	// should we process anonymous struct fields
```

Comment:
I would add some of the same warning in the go_generics main package doc to here.